recipes/assemblycomparator2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: assemblycomparator2
+  version: "2.5.6"
+
+source:
+  #url: https://github.com/cmkobel/assemblycomparator2/archive/refs/tags/v2.5.5.tar.gz [linux]
+  #md5: 38c288550d05d44e1c5c6682ecccdc23 [linux]
+  #url: https://github.com/cmkobel/assemblycomparator2/archive/refs/heads/master.zip # [linux]
+  #md5: 5c84ad46eb1c385a07332e6f9945166a # [linux]
+  url: https://github.com/cmkobel/assemblycomparator2/archive/refs/tags/v2.5.6.tar.gz
+  md5: 2ffd351078346dce8de117f973df5efc
+
+
+build:
+  number: 0
+  run_exports: snakemake
```

Comment:
Please consult the docs of using run_export at

https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#export-runtime-requirements

https://bioconda.github.io/contributor/linting.html#missing-run-exports

Essentially it's assemblycomparator2, not snakemake